src/examples/table/TableGroupedRows.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import {
+  Body,
+  Cell,
+  GroupRow,
+  Head,
+  HeaderCell,
+  HeaderRow,
+  Row,
+  Table
+} from '@zendeskgarden/react-tables';
+
+const Example = () => (
+  <div style={{ minWidth: 500 }}>
+    <Table>
+      <Head>
+        <HeaderRow>
+          <HeaderCell>Type</HeaderCell>
+          <HeaderCell>Sun exposure</HeaderCell>
+          <HeaderCell>Soil</HeaderCell>
+        </HeaderRow>
+      </Head>
+      <Body>
+        <GroupRow key="fruits">
```

Comment:
Let's make these bold please. Regular is allowed when there are key value pairs like:

> **Fruit** Plums

If there is no secondary categorization within the group row, we should use bold type.